binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="$(NuGetTargetMoniker.Contains('UAP')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
```

Comment:
I would expect that RefPath will correctly be set to `$(RefRootPath)uap` when you are in that build configuration. Is that not true?